drivers/overlay/ov_network.go

```diff
@@ -178,17 +175,88 @@ func (n *network) destroySandbox() {
 				}
 			}
 		}
+
+		if hostMode {
+			if err := removeNetworkChain(n.id[:12]); err != nil {
+				logrus.Warnf("could not remove network chain: %v", err)
+			}
+		}
+
 		sbox.Destroy()
 		n.setSandbox(nil)
 	}
 }
 
-func (n *network) initSubnetSandbox(s *subnet) error {
-	// create a bridge and vxlan device for this subnet and move it to the sandbox
-	brName, err := netutils.GenerateIfaceName("bridge", 7)
+func setHostMode() {
+	if os.Getenv("_OVERLAY_HOST_MODE") != "" {
```

Comment:
Are we expecting the users to set this environment variable on systems with kernel < 3.16 ?
I understand that checking for the kernel version may not be the best solution.
Is it possible to check for the specific kernel error that we get in kernels < 3.16 while moving the VXLAN tunnel into the namespace as the hint to set the HostMode automatically ?